js/components/controlBar.js

```diff
@@ -603,23 +466,15 @@ var ControlBar = React.createClass({
 
       volume: (
         <div key={CONSTANTS.CONTROL_BAR_KEYS.VOLUME} className="oo-volume oo-control-bar-item">
```

Comment:
If we're no longer using the class oo-control-bar-item, should we remove this one as well?

Reply:
In this case we need to set the class manually since this is not a `ControlButton`.

Reply:
Got it